etc/testing/travis.sh

```diff
@@ -2,14 +2,34 @@
 
 set -ex
 
+(
+    # Stop Travis from timing us out in 10m when we want to get retried after 20m
+    set +x
+    while true; do
+        echo "liveness ping $(date)"
+        sleep 60
+    done
+) &
```

Comment:
This seems risky to me - won't this stop all timeouts?

Reply:
Execution of `travis.sh` is wrapped in a `timeout` command, see https://github.com/pachyderm/pachyderm/pull/5137/files#diff-354f30a63fb0907d4ad57269548329e3R59